service/src/main/kotlin/fi/espoo/evaka/shared/async/AsyncJobRunner.kt

```diff
@@ -133,6 +133,7 @@ class AsyncJobRunner<T : AsyncJobPayload>(val payloadType: KClass<T>, private va
     private fun runPendingJobs(clock: EvakaClock, maxCount: Int = 1_000) {
         Database(jdbi).connect { db ->
             var remaining = maxCount
+            runningCount.incrementAndGet()
```

Comment:
The original meaning of `runningCount` was "number of currently running jobs" and now it's "number of currently active workers". I think this is fine, since we actually don't care about the former. :+1:

Reply:
Yeah, I guess it's misleading now. Would it make more sense to turn `isBusy` into `AtomicBoolean` and leave `runningCount` as is?

Reply:
`AtomicBoolean` wouldn't work, because we can have multiple workers in parallel so it wouldn't be clear which one is the last one that is allowed to set it to false. I think the `runningCount` change is ok, but maybe we could rename it to something like `activeWorkerCount` to be absolutely clear what is means. In any case it's not super important, because `runningCount` is only used in the `isBusy` getter

Reply:
Oh yeah, of course 🙈